README.md

```diff
@@ -10,7 +10,9 @@ Cryptocurrency trading bot that allows users to create strategies and then backt
 
 # Installation
 
-Please make sure you have an installation of at least Python 3.7 and are in the same directory as Algobot when running the commands below.
+Algobot requires at least Python 3.7.
+
+Once you have cloned or unpacked the source code locally. Run the following commands from your terminal in that directory.
```

Comment:
Small grammar typo, I believe it should be a comma here and uncapitalizing run :D

Reply:
Good catch ;)